novawallet/Common/Substrate/Types/CallCodingPath.swift

```diff
@@ -119,9 +119,17 @@ extension CallCodingPath {
         CallCodingPath(moduleName: "Substrate", callName: "poolSlash")
     }
 
+    static var swap: CallCodingPath {
+        CallCodingPath(moduleName: "Substrate", callName: "swap")
```

Comment:
We have concrete calls for swaps. Please, check buy/sell extrinsics we send when perform swaps